ROADMAP-TO-1.0.0.md

```diff
@@ -315,11 +265,11 @@ The libp2p protocols are the protocol stack for the modular p2p protocols librar
   - [x] Transport protocol: WebSockets in JS node
   - [x] Transport protocol: WebSockets in JS browser
   - [ ] Transport protocol: WebRTC in Go
-  - [ ] Transport protocol: WebRTC in JS node
+  - [x] Transport protocol: WebRTC in JS node
   - [x] Transport protocol: WebRTC in JS browser
   - [ ] Transport protocol: SCTP in Go
   - [ ] Transport protocol: SCTP in JS node
-  - [ ] Transport protocol: tor in Go
+  - [x] Transport protocol: tor in Go
```

Comment:
Is the onion transport safe to tick off? It isn't autited, integrated, tested, documented. Right now it's more of a proof-of-concept.